src/tools/bit2core.c

```diff
@@ -60,6 +65,37 @@ rev_info parse_revision_arg(const char* rev)
   exit(-1);
 }
 
+rev_info* find_board_revision_from_targetname(char* targetname)
```

Comment:
Maybe this should be find_fpga_part_from_targetname() instead, so that there is no ambiguity?  For example, nexys4 and nexys4ddr use the same FPGA part.

Reply:
Agree that this function name needs a re-think. My thoughts are it could change to `find_rev_info_from_fpga_target()`, as the function genuinely returns a rev_info object.

I think some of the cloudiness in the name stems from this overloaded 'target' term. In my current usage of it above, I had intended for it to mean fpga-target, but I think you interpreted it as mega65-target.

Reply:
Ok, in an upcoming commit, I've had a re-think of the terminology used within the file to be more accurate, so:
- my 'rev_info' struct got renamed to 'm65target'
- I've tried to remove the loose 'target' term and enforce either the terms 'm65target' or 'FPGA part' everywhere
- In terms of the function-name you mention here, I've thus renamed it to: **find_m65target_from_fpga_part()**.